region/tourian/main.json

```diff
@@ -1802,12 +1802,21 @@
                 {
                   "name": "Avoid Damage",
                   "notable": false,
-                  "requires": ["ScrewAttack"]
+                  "requires": [
```

Comment:
Its safe to enter not in screw, as most speedruns do.  Then use screw after

Reply:
From the left door?

Reply:
1->2 is the vertical door entry.
For the 2 door, you may have room to land first then jump.  Jumping straight in seems like you would land on the hopper to me.

Reply:
Looks like the Tourian nodes are numbered differently than most rooms. Glad you caught that. It might be worth putting them into the same order as other regions someday ...